lib/sle2docker/cli.rb

```diff
@@ -47,7 +47,7 @@ def version
     private
 
     def ensure_can_access_dockerd
-      output = `docker info`
+      output = `docker info 2>/dev/null`
```

Comment:
I don't think this is needed. It's actually helpful to check what went wrong imho